lib/identity_doc_auth/lexis_nexis/responses/lexis_nexis_response.rb

```diff
@@ -36,12 +36,23 @@ class LexisNexisResponse < IdentityDocAuth::Response
 
         def initialize(http_response)
           @http_response = http_response
-          super(
-            success: successful_result?,
-            errors: error_messages,
-            extra: extra_attributes,
-            pii_from_doc: pii_from_doc,
-          )
+
+          begin
+            super(
+              success: successful_result?,
+              errors: error_messages,
+              extra: extra_attributes,
+              pii_from_doc: pii_from_doc,
+            )
+          rescue StandardError => e
+            config.exception_notifier&.call(e)
+            super(
+              success: false,
+              errors: { network: true },
+              exception: e,
+              extra: { backtrace: e.backtrace },
+            )
+          end
```

Comment:
1. Ruby methods have implicit `begin` so we can use the top-level `rescue` and remove some indentation here
2. I think it's very strange to rescue in an initializer like this..... what's the exception come from?

```suggestion

          super(
            success: successful_result?,
            errors: error_messages,
            extra: extra_attributes,
            pii_from_doc: pii_from_doc,
          )
        rescue StandardError => e
          config.exception_notifier&.call(e)
          super(
            success: false,
            errors: { network: true },
            exception: e,
            extra: { backtrace: e.backtrace },
          )
```

Reply:
It's weird, I agree. I don't know where it will come from, it could come from the execution of any of the methods in the super() call, hence why the backstop. I keep finding unexpected responses from LN that don't conform to a consistent response object spec and we haven't even unleashed the public on it yet.

So in this case where I'm not certain what weird things we could get in the wild and an exception in this code isn't producing anything in events.log I wanted to add an explicit backstop.

Reply:
We should probably investigate why an exception isn't producing anything in events.log but at least for now with a small push  to get this into staging for testing this seemed like a weird, but not terrible way to make sure we're at least logging and surfacing unknown exceptions that may pop up. I'm fine commenting this block for removal if we find no additional errors after public release and opening a ticket to investigate why exceptions are sometimes not being logged as expected.